bolero-generator-derive/Cargo.toml

```diff
@@ -14,6 +14,7 @@ readme = "../bolero-generator/README.md"
 proc-macro = true
 
 [dependencies]
+proc-macro-crate = "1.2"
```

Comment:
I looked at how this works; quite the hack! But I'm not sure how else to accomplish it.